tcii18n/translator.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+
+class Translator:
+    def __init__(self, filename):
+        self.reader = csv.reader(open(filename, 'r'))
```

Comment:
you should close your file once it's read.

See [an example here](https://bitbucket.org/anthropedia/tci-data/src/ed0d88c6ed4995ec5195ef60974a1281b2693f67/tcidata/__init__.py?at=master&fileviewer=file-view-default#__init__.py-34)